models.py

```diff
@@ -650,11 +653,69 @@ def infer(self, encoder_outputs, Sy, B=4, debug=False, y_lengths=None):
 
 		return beam_scores, beam
 
+def obtain_glove_embeddings(filename, vocab,dim=100):
```

Comment:
For future reference, there's a package called Gensim that makes it fairly easy to load any kind of word embedding file. I use it a lot

Reply:
But this works well too 👍 , just more work for you 😄